src/Randomizer.App/RomGenerator.cs

```diff
@@ -78,12 +89,24 @@ protected byte[] GenerateRomBytes(RandomizerOptions options, out SeedData seed)
             return rom;
         }
 
+        /// <summary>
+        /// Generates a seed for a rom based on the given randomizer options
+        /// </summary>
+        /// <param name="options">The randomizer generation options</param>
+        /// <param name="seed">The string seed to use for generating the rom</param>
+        /// <returns>The seed data</returns>
         public SeedData GenerateSeed(RandomizerOptions options, string seed = null)
         {
             var config = options.ToConfig();
             return _randomizer.GenerateSeed(config, seed ?? options.SeedOptions.Seed, CancellationToken.None);
         }
 
+        /// <summary>
```

Comment:
Just as an FYI, you don't really need to add documentation to private or internal stuff, unless you think it's necessary to understand it properly.